coderd/database/migrations/000226_dormancy_notification_template.up.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO notification_templates (id, name, title_template, body_template, "group", actions)
+VALUES ('123e4567-e89b-12d3-a456-426614174000', 'Workspace Marked as Dormant', E'Workspace "{{.Labels.name}}" marked as dormant',
+        E'Hi {{.UserName}}\n\nYour workspace **{{.Labels.name}}** has been marked as dormant.\nThe specified reason was "**{{.Labels.reason}}{{ if .Labels.initiator }} ({{ .Labels.initiator }}){{end}}**',
```

Comment:
I think we'll need to add some detail here about what "dormant" means; perhaps a link to the docs and how to fix it.